packages/browserslist/source/browserslist.ts

```diff
@@ -0,0 +1,10 @@
+export const config = [
```

Comment:
@robert-adchitects @acwo 

This is a proposal for the shared configuration.
There's no rush; take your time and let me know if there's anything needed to add.

If everything is ok, please approve the Pull Request, and we will release it. 😄

Reply:
I'm approving this PR because it's blocking me from continuing with the project I have right now, and I need this package.

There is always a space to adjust the config in the next version.